plugins/experimental/maxmind_acl/mmdb.h

```diff
@@ -71,6 +66,20 @@ class Acl
     if (db_loaded) {
       MMDB_close(&_mmdb);
     }
+    for (auto &i : allow_regex) {
+      for (auto &y : i.second) {
+        if (y._rex != nullptr) {
+          delete y._rex;
+        }
+      }
+    }
+    for (auto &i : deny_regex) {
+      for (auto &y : i.second) {
+        if (y._rex != nullptr) {
+          delete y._rex;
+        }
+      }
+    }
```

Comment:
Its seems like the regex could be added to multiple or these vectors.  Is it safe to delete them repeatedly if the pointers are copied?  Seems strange.

Reply:
Could use shared_ptrs to refcount the instances, but I'm not sure how thread safe that would be.  @bryancall should Regex have a copy constructor?  Seems like somewhere else I saw needed to copy a Regex and ended up manually creating new and recompiling.

Reply:
I am also fine with just removing that loop in the regex parsing, so it would basically just take the first entry as the country code and the last entry as the regex, and skip any extra country codes that might exist. This is how its documented to work, being able to do this sharing shouldnt even technically be there anyway

Reply:
I created a PR for it #12612.  If we want it then we can merge it in.